src/wheel/cli/convert.py

```diff
@@ -293,7 +293,7 @@ def convert(files: list[str], dest_dir: str, verbose: bool) -> None:
                 source = WininstFileSource(path)
 
             if verbose:
-                print(f"{archive}... ", flush=True)
+                print(f"{archive}...", flush=True, end="")
```

Comment:
Should this be to stderr?

Reply:
I don't think so...? There's no docs indicating either way, and `wheel tags` prints the file names to stdout.